process/coordinator/process.go

```diff
@@ -1223,7 +1228,7 @@ func (tc *transactionCoordinator) handleProcessTransactionError(snapshot int, mi
 
 	err := tc.accounts.RevertToSnapshot(snapshot)
 	if err != nil {
-		// TODO: evaluate if reloading the trie from disk will might solve the problem
+		// TODO: evaluate if reloading the trie from disk might solve the problem
```

Comment:
I think we can remove this TODO completely. If we fail to do a revert on the trie, most certainly will get a StateRootHashMismatch error and the block will be reverted causing the automatic reload of the trie in that case and the block processing retrial.

Reply:
done